src/prompt.c

```diff
@@ -88,3 +93,38 @@ void	prompt_destroy(t_prompt *_prompt)
 		free(_prompt->buffer);
 	free(_prompt);
 }
+
+void	prompt_update(t_prompt *_prompt)
+{
+	prompt_update_current_path(_prompt);
+	prompt_prepare_buffer(_prompt);
+}
+
+char	*prompt_read(void)
+{
+	char	*command_line;
+
+	if (!(get_next_line(STDIN, &command_line) > 0))
+		return (NULL);
+	return (command_line);
+}
+
+void	prompt_loop(t_prompt *_prompt)
+{
+	char	*command_line;
+	
+	command_line = NULL;
+	while(1)
+	{
+		prompt_update(_prompt);
+		prompt_print(_prompt);
+		command_line = prompt_read();
+		if (ft_strcmp(command_line, "exit") == 0)
```

Comment:
what if you would add 'break' as well? what if there is a space in front of ' exit'? why not use the lexer/parser we use for all other commands?